Classes/Controller/FlickrController.php

```diff
@@ -0,0 +1,81 @@
+<?php
+namespace Neos\Demo\Controller;
+
+/*
+ * This file is part of the Neos.Demo package.
+ *
+ * (c) Contributors of the Neos Project - www.neos.io
+ *
+ * This package is Open Source Software. For the full copyright and license
+ * information, please view the LICENSE file which was distributed with this
+ * source code.
+ */
+
+use Neos\Flow\Annotations as Flow;
+use Neos\Flow\Mvc\Controller\ActionController;
+
+/**
+ * Controller that displays flickr photo streams
+ */
+class FlickrController extends ActionController
+{
+    /**
+     * @Flow\InjectConfiguration(path="flickr.tagStreamUriPattern")
+     * @var string
+     */
+    protected $tagStreamUriPattern;
+
+    /**
+     * @Flow\InjectConfiguration(path="flickr.userStreamUriPattern")
+     * @var string
+     */
+    protected $userStreamUriPattern;
+
+    /**
+     * @return void|string
+     */
+    public function tagStreamAction()
+    {
+        $tags = $this->request->getInternalArgument('__tags') ?: 'default-tag'; // todo how to pass `__tags` from node?
```

Comment:
why is this not working when being set from the inspector?

<img width="167" alt="image" src="https://github.com/neos/Neos.Demo/assets/85400359/9520a308-0a42-4636-afa0-2d39a31cc0c9">

Reply:
ha something we dint broke .. at least not with 9.0 it seems in 7.3 that feature doesnt work as well....

Reply:
the internal arguments is everything given via fusion, I think WAY back in times of "magic" prototypes from processors, the plugin processing would expose all properties as internal arguments, but since that is gone you need to either go from the node (which is there by default) or add the properties by hand (basically any property ON the plugin prototype is exposed as internal argument to the controller)

Reply:
yes okay that works now by adding `tags = ${q(node).property('tags')}`